cardInfoAPI-Level1.yaml

```diff
@@ -0,0 +1,1402 @@
+          name: card_account_reference
+          required: false
+          schema:
+            $ref: '#/components/schemas/CardAccountReference'
+          description: Retrieves all cards related to a card account (not bank account). Value has to be a functional reference to the account that can be known by the end customer.
+        - in: query
+          name: bank_account_reference
+          required: false
+          schema:
+            $ref: '#/components/schemas/BankAccountReference'
+          description: Retrieves all cards related to a bank account (not card account). Value has to be a functional reference to the account that can be known by the end customer and is usually an IBAN.
+        - $ref: '#/components/parameters/limit'
+        - $ref: '#/components/parameters/offset'
+        - $ref: '#/components/parameters/client_in_header'
+        - $ref: '#/components/parameters/correlation_in_header'
+        - $ref: '#/components/parameters/agent_in_header'
+      responses:
+        '200':
+          description: Payment cards were found which match the query parameters. If no cards matched the parameters or access rights are missing, an empty list is returned.
+          headers:
+            X-Correlation-ID:
+              $ref: '#/components/headers/X-Correlation-ID'
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  cardList:
+                    type: array
+                    items:
+                      $ref: '#/components/schemas/CardDetails'
+                  _links:
+                    $ref: '#/components/schemas/LinksPagination'
+        '400':
+          $ref: '#/components/responses/standard400'
+        '500':
+          $ref: '#/components/responses/standard500'
+
+  /cards/{cardId}:
+    get:
+      summary: Provides a single card which matches the id.
+      description: >
+        This operation enables the retrieval of a single payment card when the cardId is already known.
+        The cardId must be retrieved with the GET /cards operation before or stored locally at the API consumer.
+      tags:
+        - Cards
+      parameters:
+        - $ref: '#/components/parameters/path_cardId'
+        - $ref: '#/components/parameters/client_in_header'
+        - $ref: '#/components/parameters/correlation_in_header'
+        - $ref: '#/components/parameters/agent_in_header'
+      responses:
+        '200':
+          description: Single payment card was found. If the cardId was not found, a 400 is returned.
+          headers:
+            X-Correlation-ID:
+              $ref: '#/components/headers/X-Correlation-ID'
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/CardDetails'
+        '400':
+          $ref: '#/components/responses/standard400'
+        '500':
+          $ref: '#/components/responses/standard500'
+
+  /card-tokens:
```

Comment:
Use Case unclear. which card tokens are appropiate (wallets on client side or wallets in merchant stores)?

Reply:
Both types of tokens will be returned.

Reply:
It is great that both cases are supported. Nethertheless, it is needed that only a cardholder can access to such data.

Reply:
Agreed that data can only be accessed according to the consent granted by the cardholder.